src/main/java/com/mraof/minestuck/tileentity/redstone/WirelessRedstoneTransmitterTileEntity.java

```diff
@@ -127,10 +153,18 @@ public void load(BlockState state, CompoundNBT compound)
 		super.load(state, compound);
 		
 		tickCycle = compound.getInt("tickCycle");
-		int destX = compound.getInt("destX");
-		int destY = compound.getInt("destY");
-		int destZ = compound.getInt("destZ");
-		this.destBlockPos = new BlockPos(destX, destY, destZ);
+		int offsetX = compound.getInt("offsetX");
+		int offsetY = compound.getInt("offsetY");
+		int offsetZ = compound.getInt("offsetZ");
+		this.offsetPos = new BlockPos(offsetX, offsetY, offsetZ);
```

Comment:
Could you move offset loading into an if-else statement with the dest loading?
It makes it clearer that dest coords is an alternative format to offset coords if you write it like this:
```java
if (should load offset coords)
{
	load offset;
} else
{
	load dest;
}
```
or
```java
if (should load dest coords)
{
	load dest;
} else
{
	load offset;
}
```

Reply:
Moved it around so that it checks for the absolute coordinates first and then put the offset in the else statement